addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComBBQTemperatureMessage.java

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.smarthome.core.library.items.NumberItem;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComUnsupportedValueException;
+
+/**
+ * RFXCOM data class for BBQ temperature message.
+ *
+ * @author Mike Jagdis - Initial contribution
+ */
+public class RFXComBBQTemperatureMessage extends RFXComBaseMessage {
```

Comment:
Because there already was a "bbqtemperature" thing name defined (even if it wasn't used) and because the message is very specifically BBQ temperatures only. It's not inconceivable that there might one day be a BBQControl too. Er... not _quite_ inconceivable anyway...

I know there was already a packet type of BBQ1 defined in a few places, including openhab1. But nothing was ever actually implemented and the packet type is actually BBQ - BBQ1 is a subtype (actually the only supported subtype). So there was nothing of value to preserve.